nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -216,6 +220,30 @@ public static void main(String[] args) throws IOException, InterruptedException
                 dumpFile = null;
                 verbose = false;
             }
+        } else if (cmd.equalsIgnoreCase("status-history")) {
+            if (args.length != 4) {
+                System.err.printf("Wrong number of arguments: %d instead of 4, the command parameters are: " +
+                        "status-history --days <number of days> <dumpFile>%n", args.length);
+                System.exit(0);
+            }
+            statusHistoryDays = args[2];
+            try {
+                final int numberOfDays = Integer.parseInt(statusHistoryDays);
+                if (numberOfDays < 1) {
+                    System.err.println("The number of days must be positive and greater than zero.");
+                    System.exit(0);
```

Comment:
Exit 0 is for cases where there is no issue. It would be better to use some error codes